include/calculate/symbol.hpp

```diff
@@ -47,8 +47,8 @@ class Symbol {
 public:
     template<
         typename Callable,
-        std::enable_if_t<Inspect<Callable>::not_me>* = nullptr,
-        std::enable_if_t<!Inspect<Callable>::is_model>* = nullptr
+        typename = std::enable_if_t<Inspect<Callable>::not_self>,

```

Comment:
I believe this is a little more readable use of [SFINAE](http://en.cppreference.com/w/cpp/language/sfinae).